signalfx-tracing/signalfx-nodejs-tracing/mysql/deedScheduler/scheduler.js

```diff
@@ -0,0 +1,98 @@
+const tracer = require('./tracer');
+
+const dbOption = require('./config').mysqlVersion;
+
+let myDB;
+
+if (dbOption == 1) {
+  myDB = require('./dbDriver/mysql_driver');
+} else {
+  myDB = require('./dbDriver/mysql2_driver');
+}
+
+
+async function addItem(deed, note, day) {
+  const span = tracer.scope().active();
+  span.setTag('addedDeed', deed);
+  let response;
+  await myDB.addItem(deed, note, day)
+      .then(
+          (rows) => {
+            response = rows;
+          },
+          (error) => {
+            console.log(error);
+          });
+  return (response);
+}
+
+
+async function deleteDeed(deed, day) {
+  const span = tracer.scope().active();
+  span.setTag('deletedDeed', deed);
+  let response;
+  await myDB.deleteDeed(deed, day)
+      .then(
+          (rows) => {
+            response = rows;
+          },
+          (error) => {
+            console.log(error);
+          });
+  return response;
+}
+
+
+async function listDeeds(day) {
+  const span = tracer.scope().active();
+  span.setTag('sch_list', true);
```

Comment:
`sch_list`?

Reply:
`scheduler_list` (will remove. Was part of me trying to follow the function calls in the traces)